main/nav_create_simulated_model.py

```
@@ -1325,6 +1416,94 @@ def _on_star_field(self, idx: int, key: str, value: Any) -> None:
             self._updater.request_update()
 
+    def _on_star_psf_size_v_slider(self, idx: int, value: int) -> None:
+        # Convert slider position (0-11) to odd value (1, 3, 5, ..., 23)
+        odd_value = value * 2 + 1
+        tab_idx = self._find_tab_by_properties('star', idx)
+        if tab_idx is not None:
+            tab_w = self._tabs.widget(tab_idx)
+            if tab_w is not None:
+                spin = getattr(tab_w, 'psf_size_v_spin', None)
+                if spin is not None:
+                    spin.blockSignals(True)
```

Ruff (0.14.7)

Boolean positional value in function call

(FBT003)